apps/air-discount-scheme/web/screens/Admin/Admin.tsx

```diff
@@ -76,14 +75,7 @@ const Admin: Screen = ({}) => {
   } as any)
   const input = {
     ...filters,
-    airline:
-      filters.airline?.value === Airlines.norlandair
-        ? Airlines.icelandair
-        : filters.airline?.value,
-    cooperation:
-      filters.airline?.value === Airlines.norlandair
-        ? Airlines.norlandair
-        : undefined,
```

Comment:
Then this needs to be reverted and maybe just add a Selection field that contains a cooperation. Doesn't need the airline trick